ineffassign.go

```diff
@@ -13,9 +13,10 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/gordonklaus/ineffassign/pkg/ineffassign"
 	"golang.org/x/tools/go/analysis"
 	"golang.org/x/tools/go/packages"
+
+	"github.com/gordonklaus/ineffassign/pkg/ineffassign"
```

Comment:
apparently my goimports order is this 🤔 
happy to undo if you have any preference